### ISSUE TITLE
feat req: show password on update

### ISSUE THREAD

**Issue by chrillep** [oooholdings/nova-password-generator#20]
i would like to have the same features on update as on detail and index.
We are using a settings plugin so the only view available is always update/save.
but basicly

```php
PasswordGenerator::make( 'Personal Access Token' )
    ->showValueOnUpdate( bool $show = true )
    ->blurValueOnUpdate( bool $show = true )
    ->redactValueOnUpdate( bool $redact = true, string $character = '•' )
```

**Comment by btrsco** [comment]
@chrillep This sounds like a good idea, I just have one question about the implementation. Are you expecting to use this field the same way as the details/index page (basically readonly) or still be able to edit and show the generator UI?

**Comment by chrillep** [comment]
ooh yeah sry. meant a mix of create and detail. 
as you said. 

> still be able to edit the value and show the generator UI.

**Comment by chrillep** [comment]
bt love the implementation!

**Comment by chrillep** [comment]
is it this part?
https://github.com/oooholdings/nova-password-generator/blob/2b350ab763e4e17857a10a16ac790dec8c6d2e7e/resources/js/components/FormField.vue#L14
that should be
```vue
v-model="field.value" /> 
```

?

**Comment by chrillep** [comment]
is it this part?

https://github.com/oooholdings/nova-password-generator/blob/2b350ab763e4e17857a10a16ac790dec8c6d2e7e/resources/js/components/FormField.vue#L168-L170

changed to

```js
        setInitialValue() {
            this.value = this.field.value;
        },
```

**Comment by btrsco** [comment]
Sorry for the delay, I just started a new job and have haven't had much free time.

So I have looked over your pull request and am confused as to what exactly you're asking for. Can you please explain your use-case a little more? Seems like you're using a Settings package and that page mimics the form pages all the time. Do you just want the value to be filled automatically instead of being empty or is there more to this specific feature request?

**Comment by FreekVR** [comment]
I have an (I think) somewhat similar request though I am not sure the PR covers that.

We are using the password generator to generate an optional API secret attached to a specific model. That secret can be used by external services to access an API allowing for mutations to some related model.

We'd like to show this secret (which we store as plain text) on the edit page as well, so it's clear from that context if one already exists so that the user doesn't accidentally overwrite an existing secret.

**Comment by chrillep** [comment]
> Sorry for the delay, I just started a new job and have haven't had much free time.
> 
> 
> 
> So I have looked over your pull request and am confused as to what exactly you're asking for. Can you please explain your use-case a little more? Seems like you're using a Settings package and that page mimics the form pages all the time. Do you just want the value to be filled automatically instead of being empty or is there more to this specific feature request?

Your spot on . 

We want the value to be filled automatically instead of being empty.

**Comment by btrsco** [comment]
Thank you for the feature request @chrillep, again sorry it took so long to get to this. This has been implemented in [v1.8.0](https://github.com/oooholdings/nova-password-generator/releases/tag/v1.8.0)

**Comment by chrillep** [comment]
> Thank you for the feature request @chrillep, again sorry it took so long to get to this. This has been implemented in [v1.8.0](https://github.com/oooholdings/nova-password-generator/releases/tag/v1.8.0)

no worries just appreciate the work u put in!
